### PR TITLE
Enable PHP-CS-Fixer in Github Action

### PR DESCRIPTION
**Pull request by lucasmichot**
This PR enables PHP-CS-Fixer in Github Action.
The coding rules and ruleset are way more flexible than in StyleCI.

To fix the violations, just run `composer lint`

---

Example of failing linting: https://github.com/osiset/laravel-shopify/pull/883/checks?check_run_id=3030951877

---

651a7f679ace85c2767746a3a36fe16203055d0f fix all code violations

---

Of course, StyleCI service needs to be disabled.

### CODE REVIEW

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    'yoda_style' => [
+        'always_move_variable' => false,
+        'equal' => false,
+        'identical' => false,
+    ],
+];
+
+$finder = Finder::create()->in(__DIR__)
+    ->notPath([
+        'src/ShopifyApp/resources/config/shopify-app.php',
```
Comment (lucasmichot) [inline .php-cs-fixer.dist.php:57]: The config file is ignored, on purpose.

**Comment by osiset** [comment]
@lucasmichot is this in draft until I disable styleci? I can disable it if need be.

**Comment by lucasmichot** [comment]
> @lucasmichot is this in draft until I disable styleci? I can disable it if need be.

Yes StyleCI needs to be disabled 👍

**Comment by osiset** [comment]
Disabled StyleCi

**Pull request merged**